packages/chakra-ui-core/src/CAspectRatioBox/tests/CAspectRatioBox.test.js

```diff
@@ -0,0 +1,48 @@
+import { CAspectRatioBox, CBox } from '../..'
+import { render } from '@/tests/test-utils'
+
+jest.mock('breadstick/dist/components/Alert/styles.css', () => ({})) // jest tries to import styles and fails...
```

Comment:
Is there a global setup we can use to mock this library with jest? Feels a bit weird having it most test files 🤔 Will look into this to share my findings as well.

Reply:
yes we can